core/validation/initial_validation/schemas.py

```diff
@@ -63,18 +63,18 @@
         column=5,
         row=4,
         expected_values=(),
-        error_message="You’re not authorised to submit for {wrong_place_or_fund_type}. You can only submit for "
-        "{allowed_places_or_fund_types}.",
+        error_message="You’re not authorised to submit for {entered_value}. You can only submit for "
+        "{allowed_values}.",
         auth_type="Fund Types",
     ),
     AuthorisationCheck(
         sheet="2 - Project Admin",
         column=6,
         row=4,
         expected_values=(),
-        error_message="You’re not authorised to submit for {wrong_place_or_fund_type}. You can only "
-        "submit for {allowed_places_or_fund_types}.",
-        auth_type="Place Names",
+        error_message="You’re not authorised to submit for {entered_value}. You can only "
+        "submit for {allowed_values}.",
+        auth_type="Programme",
```

Comment:
We need to keep the key for TFs programe auth as `Place Names` because otherwise integration with the front-end will break atm. I think this should be changed in a later PR, after a change to the front-end to allow for this to not be a breaking change.